src/exp.jl

```diff
@@ -192,6 +192,36 @@ Base.cis(A::Union{SkewHermitian,SkewHermTridiagonal}) = skewcis!(copyeigtype(A))
 Base.cos(A::Union{SkewHermitian,SkewHermTridiagonal}) = skewcos!(copyeigtype(A))
 Base.sin(A::Union{SkewHermitian,SkewHermTridiagonal}) = skewsin!(copyeigtype(A))
 
+@views function skewsincos!(A::Union{SkewHermitian{<:Real},SkewHermTridiagonal{<:Real}})
+    n = size(A,1)
+    if typeof(A) <:SkewHermitian
+        n == 1 && return fill(0, 1, 1), Symmetric(fill(1, 1, 1))
```

Comment:
This is not type-stable: `fill(0, 1, 1)` etcetera always returns an array of `Int`.

Reply:
I will cast returned values to the right type